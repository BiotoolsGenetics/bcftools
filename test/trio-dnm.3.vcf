##fileformat=VCFv4.2
##FORMAT=<ID=AD,Number=.,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
##FORMAT=<ID=AQ,Number=R,Type=Integer,Description="Phred-score allele quality sum for +trio-dnm calling">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification">
##contig=<ID=1,length=249250621>
##reference=file:///lustre/scratch113/resources/ref/Homo_sapiens/1000Genomes_hs37d5/hs37d5.fa
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	proband	father	mother
1	3	.	G	A,T	.	.	.	GT:PL:AD:AQ	0/1:255,0,255,255,255,255:37,46,0:1261,1641,0	0/0:0,193,255,193,255,255:64,0,0:2143,0,0	0/0:0,199,255,199,255,255:66,0,0:2201,0,0
