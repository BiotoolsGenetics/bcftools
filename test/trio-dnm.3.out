1	255	.	.		55	0	0
